scopeline.el

```diff
@@ -77,7 +77,9 @@
     (js-mode . ("function" "function_declaration" "if_statement" "for_statement" "while_statement"))
     (js2-mode . ("function" "function_declaration" "if_statement" "for_statement" "while_statement"))
     (js3-mode . ("function" "function_declaration" "if_statement" "for_statement" "while_statement"))
+    (typescript-mode . ("function" "function_declaration" "if_statement" "for_statement" "while_statement"))
     (json-mode . ("pair"))
+    (toml-mode . ("pair"))
```

Comment:
For toml, I think we would ideally need `table` and `table_array_element` as well, but it currently shows up along with the next one (because the end of first is on the first char before the start of next on[curent cursor position]). Let me merge this in for now though. Thanks.

![2023-03-14-09-35-01](https://user-images.githubusercontent.com/14259816/224890429-e1963f5a-eee8-4d36-9739-51f268b358f8.png)